### DISCUSSION TITLE
Cycle Computing

### DISCUSSION THREAD

**Discussion by 6cs4u** [Anura3D/Anura3D_OpenSource#166] [Q&A]
I tried using Anura3D for loop calculation, but encountered this problem during the second calculation. How can I solve it. Thank you for your reply.
![屏幕截图 2023-12-10 155403](https://github.com/Anura3D/Anura3D_OpenSource/assets/145540472/19ee1b7d-4ef6-4f22-8764-21dc7a7a634f)

**Comment by murphy-john** [comment]
Will you provide additional information about you model? Please follow [the posting guidelines here](https://github.com/Anura3D/Anura3D_OpenSource/discussions/29#discussion-4274028). What is a loop calculation? 

Are you running your model on a portable hard drive? That has caused issues for other users. Do you have read & write permissions to the folder where you are running your model?

**Reply by 6cs4u** [reply]
Sorry, I didn't describe my problem clearly before. I am trying to develop Anura3D for slope failure probability calculation, and I need to calculate the model many times, so I wrote a loop to repeatedly call the Anura3D program. The program ran normally during the first calculation, but this issue occurred during the second calculation. I would like to ask if there is a good way to have the Anura3D program repeat the call. Thank you for your reply.

**Reply by murphy-john** [reply]
Are you trying to do a parametric analysis? 

If so, I think you should have your script create a new folder for each "cycle". Then edit the parameters you want and run the code in that new folder. 

My guess is the code is already seeing the files created from the first "cycle". The code gets confused because they shouldn't exist for the "new" model in the second "cycle".
